### PR TITLE
Allow to save raw quantification before normalisation

### PR DESCRIPTION
**Pull request by nh3**

### CODE REVIEW

File: bin/scanpy-normalise-data.py [hunk 1]
```diff
@@ -27,6 +30,10 @@ def main(args):
                            type=float,
                            default=1e4,
                            help='Aimed counts per cell after normalisation, default: 1e4')
+    argparser.add_argument('-r', '--save-raw',
```
Comment (pinin4fjords) [inline bin/scanpy-normalise-data.py]: Why log scale? Conventionally (at least in R world), raw data is kept untransformed.

Comment (nh3) [reply bin/scanpy-normalise-data.py]: It's log1p, so it's reversible. Anyway, this is what the scanpy tutorial is doing:

> Set the .raw attribute of AnnData object to the logarithmized raw gene expression for later use in differential testing and visualizations of gene expression. This simply freezes the state of the AnnData object returned by sc.pp.log1p.
> 
> `adata.raw = sc.pp.log1p(adata, copy=True)`

Comment (pinin4fjords) [reply bin/scanpy-normalise-data.py]: Okay, fair enough.

**Comment by pinin4fjords** [comment]
@nh3 in future please do be more descriptive with PRs- i.e. populate the description and describe what the PR is doing in qualitative terms. Will make review quicker :-)

**Pull request merged**